Making Filesystem Operations Connection Consistent

wjohnson opened the pull request:
Making Filesystem Operations Connection Consistent With Account Management syntax.

azurecla commented:
Hi __@scepas__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tysonn commented:
@scepas - Thanks for the fixes

wjohnson committed:
Making Filesystem Operations Connection Consistent

Making Filesystem Operations Connection Consistent With Account Management syntax.
